recipe/meta.yaml

```
@@ -40,58 +40,61 @@ outputs:
       run:
         - python >=3.8
 
-        # dependencies matched to pip
+        # dependencies matched to setup.cfg
+        - app-model>=0.1.0,<0.3.0a
         - appdirs >=1.4.4
         - cachey >=0.2.1
         - certifi >=2020.6.20
         - dask >=2.15.0,!=2.28.0
-        - imageio >=2.5.0,!=2.11.0
+        - imageio>=2.5.0,!=2.11.0,!=2.22.1
         - jsonschema >=3.2.0
         - magicgui >=0.3.6
-        - napari-console >=0.0.4
+        - napari-console >=0.0.6
         - napari-plugin-engine >=0.1.9
-        - napari-svg >=0.1.4
-        - npe2 >=0.3.0
+        - napari-svg >=0.1.6
+        - npe2 >=0.5.2
         - numpy >=1.18.5
         - numpydoc >=0.9.2
         - pandas >=1.3.0
         - pillow !=7.1.0,!=7.1.1
         - pint >=0.17
         - psutil >=5.0
-        - pyopengl >=3.1.0
-        - pyyaml >=5.1
+        - psygnal>=0.3.4
         - pydantic >=1.9.0
         - pygments >=2.4.0
-        - qtpy >=1.7.0
-        - scipy >=1.5.4
+        - pyopengl >=3.1.0
+        - pyyaml >=5.1
+        - qtpy >=1.10.0
         - scikit-image >=0.19.1
-        - superqt >=0.2.5
+        - scipy >=1.5.4
+        - sphinx <5
+        - superqt >=0.3.0
         - tifffile >=2020.2.16
-        - typing_extensions
         - toolz >=0.10.0
         - tqdm >=4.56.0
-        - vispy >=0.10,<0.11
+        - typing_extensions
+        - vispy >=0.11.0,<0.12.0a
         - wrapt >=1.11.1
 
         # additional dependencies for convenience in conda-forge
         - fsspec
-        - scikit-image >=0.18.1,!=0.19.0
-        - pooch >=1.3.0
+        - pooch >=1.6.0
```

From v0.4.17's setup.cfg: https://github.com/napari/napari/blob/878f1233e3c0b40afef062e711d8238fe4b44b84/setup.cfg#L116

```suggestion
        - pooch >=1.3.0
```